tests/conftest.py

```diff
@@ -1,7 +1,10 @@
 import pytest
```

Comment:
Also, we seem to have got a CI pre-commit error. It's not from anything you have touched (it's somehow slipped through the CI), but it'll need fixing before this will build properly. Running pre-commit locally should sort it out